### PR TITLE
[13829 ] add non-root users to the softhsm2 group

### PR DESCRIPTION
**Pull request by MiguelBarro**
Include an explanation of the softhsm2 group on linux

### CODE REVIEW

File: docs/installation/sources/sources_linux.rst [hunk 1]
```diff
@@ -136,6 +136,13 @@ For example, on Ubuntu use the command:
 
    sudo apt install softhsm2
 
+Note that the *softhsm2* package creates a new group called *softhsm2*. In order
```
Comment (jsantiago-eProsima) [inline docs/installation/sources/sources_linux.rst:139]:
```suggestion
Note that the *softhsm* package creates a new group called *softhsm*. In order
```

**Review by jsantiago-eProsima — CHANGES_REQUESTED** [review]

**Review by jsantiago-eProsima — DISMISSED** [review]
LGTM

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM

**Comment by jsantiago-eProsima** [comment]
@richiprosima please test this

**Pull request merged**